tests/perf/prerequisites/istio/setup.sh

```diff
@@ -0,0 +1,57 @@
+#!/usr/bin/env bash
+
+set -e
+
+WORKING_DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" >/dev/null 2>&1 && pwd )"
+
+if [[ -z "${NAMESPACE}" ]]; then
+  export NAMESPACE=istio-perf-test
+fi
+
+CLUSTER_DOMAIN_NAME=$(kubectl get gateways.networking.istio.io kyma-gateway -n kyma-system -ojsonpath="{.spec.servers[0].hosts[0]}" | sed 's/*.//g' )
+export CLUSTER_DOMAIN_NAME
+
+export WORKLOAD_SIZE=${WORKLOAD_SIZE:-10}
+export VUS=${VUS:-64}
+
+common_resources=(
+    namespace.yaml
+)
+
+workload_resources=(
+    app.yaml
+    api.yaml
+)
+
+for resource in "${common_resources[@]}"; do
+    envsubst <"${WORKING_DIR}/$resource" | kubectl -n "${NAMESPACE}" apply -f -
+done
+
+for (( i = 0; i < WORKLOAD_SIZE; i++ )); do
+    export WORKER=$((i + 1))
+    for resource in "${workload_resources[@]}"; do
+        echo "setting up resource: ${resource} for worker ${WORKER}"
+        envsubst <"${WORKING_DIR}/$resource" | kubectl -n "${NAMESPACE}" apply -f -
+
+        LIMIT=15
+        COUNTER=0
+        SUCCESS="false"
+
+        while [ ${COUNTER} -lt ${LIMIT} ]; do
```

Comment:
We are waiting for every pod before we start the next one. Is that on purpose? What if we start all pods at once, then wait for their readiness? A cluster can start pods in parallel, this could reduce setup time significantly